src/select.coffee

```diff
@@ -80,8 +81,19 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
   # we need to set tabindex so that div responds to key events
   tabindex: 0
 
+  ###*
+  * Flag controlling the dropdown's appearance of the select component
```

Comment:
This comment is a little confusing to me. I think it should be something like "Flag that determines whether or not the dropdown is shown".